great_expectations_cloud/agent/actions/__init__.py

```diff
@@ -10,6 +10,9 @@
 from great_expectations_cloud.agent.actions.draft_datasource_config_action import (
     DraftDatasourceConfigAction,
 )
+from great_expectations_cloud.agent.actions.generate_schema_change_expectations_action import (
```

Comment:
sorry I feel like this should have been included in a previous PR :) Thanks for catching.